build.gradle.kts

```diff
@@ -637,13 +649,7 @@ task("runPrettyPrintersTests") {
             "cmd /C set PYTHONIOENCODING=utf8 & $runCommand".execute("pretty_printers_tests")
         } else {
             // TODO: Remove after CLion snapshot builds provide these files with required permissions
```

Comment:
It seems this comment should be removed as well

Reply:
It's still relevant because we still have to invoke `setExecutable` on macOS

Reply:
Oh, I see